src/cljam/util/chromosome.clj

```diff
@@ -32,7 +37,14 @@
 (defn normalize-chromosome-key
   "Normalizes chromosome name."
   [s]
-  (-> s
-      normalize-name
-      prepend-chromosome-prefix
-      normalize-chromosome-prefix))
+  (let [[base version-suffix] (split-version-suffix s)]
+    (str (-> base
+             normalize-name
+             prepend-chromosome-prefix
+             normalize-chromosome-prefix)
+         (if version-suffix (cstr/lower-case version-suffix)))))
+
+(defn is-primary-chromosome?
+  [s]
+  (not (nil? (re-matches #"^chr([0-9]{1,2}|X|Y|M|MT)$"
```

Comment:
`not nil?` can be replaced with `some?`, and `^$` is not needed for `re-matches`.